exp/operator/main.go

```diff
@@ -17,69 +17,176 @@ limitations under the License.
 package main
 
 import (
+	"context"
 	"flag"
+	"fmt"
+	"math/rand"
+	"net/http"
 	"os"
+	"time"
 
+	"github.com/spf13/pflag"
+	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	"k8s.io/klog"
 	"k8s.io/klog/klogr"
-	ctrl "sigs.k8s.io/controller-runtime"
-
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha4"
 	operatorv1alpha4 "sigs.k8s.io/cluster-api/exp/operator/api/v1alpha4"
 	"sigs.k8s.io/cluster-api/exp/operator/controllers"
+	"sigs.k8s.io/cluster-api/feature"
 	"sigs.k8s.io/cluster-api/version"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/healthz"
 	// +kubebuilder:scaffold:imports
 )
 
 var (
 	scheme   = runtime.NewScheme()
 	setupLog = ctrl.Log.WithName("setup")
+
+	// flags
+	metricsBindAddr             string
+	enableLeaderElection        bool
+	leaderElectionLeaseDuration time.Duration
+	leaderElectionRenewDeadline time.Duration
+	leaderElectionRetryPeriod   time.Duration
+	watchNamespace              string
```

Comment:
I don't remember well from the proposal, but if the operator resources are going to be created in the same namespace where the provider are deployed, I'm not sure it makes sense to have this flag.
However, I'm not opposed to leave it given that it defaults to empty

Reply:
Can we leave it for now?

Reply:
It's usually better to add than remove (removals are braking changes and harder to do)

Reply:
👍 I removed it